module/localisation/OdometryLocalisation/src/OdometryLocalisation.cpp

```diff
@@ -40,16 +42,21 @@ namespace localisation {
 
 
         on<Trigger<Sensors>, Sync<OdometryLocalisation>, Single>().then("Odometry Loc", [this](const Sensors& sensors){
-        	
+
         	Transform2D Trw = Transform3D(convert<double, 4, 4>(sensors.world)).projectTo2D();
         	Transform2D Twr = Trw.i();
-        	
-            if(arma::norm(Twr.localToWorld(Trw)) > 0.00001){
-                log("arma::norm(Twr.localToWorld(Trw))",Trw.t(),Twr.t(),Twr.localToWorld(Trw).t());
-            }
-            if(arma::norm(Trw.localToWorld(Twr)) > 0.00001){
-                log("arma::norm(Trw.localToWorld(Twr))",Trw.t(),Twr.t(),Trw.localToWorld(Twr).t());
-            }
+
```

Comment:
Should this stuff really be part of this PR? And if you're deleting it delete it.